server/athenian/api/internal/features/entries.py

```diff
@@ -1450,3 +1545,32 @@ def make_calculator(
     return MetricEntriesCalculator(
         account, meta_ids, DEFAULT_QUANTILE_STRIDE, mdb, pdb, rdb, cache,
     )
+
+
+@dataclass(frozen=True, slots=True)
+class PullRequestMetricsLineRequest:
+    """Information for a single pull request metrics request."""
+
+    metrics: Sequence[str]
+    time_intervals: Sequence[Sequence[datetime]]
+    lines: Sequence[int]
+    environments: Sequence[str]
+    repositories: Sequence[Collection[str]]
+    participants: list[PRParticipants]
+
+    def __str__(self) -> str:
```

Comment:
Can you please also define `__sentry_repr__` with basic stats - how many metrics, repos, participants, etc. in each group.

Reply:
done in the same style of `PRDataFrames`